templates/admin/tool_shed_repository/common.mako

```diff
@@ -98,19 +115,21 @@
             revision_label_str = ' '
     %>
     <div class="form-row">
+        <p>By default Galaxy will install all needed dependencies for${revision_label_str}the repository. See the <a target="_blank" href="https://docs.galaxyproject.org/en/master/admin/dependency_resolvers.html">dependency resolver documentation</a>.</p>
+        <p>You can fine control the installation if you know what you are doing. <button class="toggle-detail-section">Display Details</button></p>
```

Comment:
How about "You can control how dependencies are installed (this is an advanced option, if in doubt, use the default)"